apps/evans/components/pages/document/ListenBlock.tsx

```diff
@@ -38,46 +41,31 @@ const ListenBlock: React.FC<Props> = ({ title, numAudioParts, embedId, ...props
       )}
     >
       <DownloadAudiobook
-        className="mb-8 sm:mb-16 lg:border lg:border-l-0 lg:-mt-12 lg:pt-12 lg:px-12 border-flgray-200 xl:mr-6"
+        className="mb-8 sm:mb-16 lg:border lg:border-l-0 lg:-mt-12 lg:pt-12 lg:px-12 border-flgray-200 xl:mr-6 shrink-0"
         {...props}
         quality={quality}
         setQuality={setQuality}
       />
-      <div className="flex-grow lg:ml-8 xl:max-w-screen-md xl:mx-auto">
+      <div className="flex-grow lg:max-w-[480px] min-[1140px]:max-w-xl min-[1340px]:max-w-3xl lg:mx-auto flex flex-col">
```

Comment:
i noticed at 1141 px wide, things don't look so good currently. both the audio and the headline are smashed up against the left side download block.